client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/object/ClassToSchema.java

```diff
@@ -67,6 +67,8 @@ public class ClassToSchema {
 
     private static final String fieldRefPostfix = "\"}";
 
+    private static boolean objectFieldsRequired = false;
```

Comment:
this creates a mutable static state. those must be avoided, due to lot of issues they bring. see `docs/for_developers.md` for more info on this. should rather be passed as input to `registerSchemaIfNeeded`